physics/SFC_Models/Land/Noahmp/noahmpdrv.F90

```diff
@@ -986,7 +988,7 @@ subroutine noahmpdrv_run                                       &
 !
 !  --- Just return if external land component is activated for two-way interaction
 !
-  if (cpllnd .and. cpllnd2atm) return
+  if (cpllnd .and. cpllnd2atm .and. (.not. flag_init)) return
```

Comment:
@uturuncoglu I'm a little confused at the logic now. I thought with the new approach that we wouldn't need to call noahmpdrv_run at all. Isn't this saying that with the land component, on the first time step we need to run the ccpp noahmp?

Reply:
that was my main point, that the changes to noahmpdrv.F90 and .meta don't seem necessary anymore.